index.html

```diff
@@ -39,6 +39,190 @@ <h2 class="connect">
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- Third card -->
+
+        <div class="work-container">
+
+            <div class="project1">
+                <img src="images/tonic-image-2.png" alt="Snapshoot Portfolio" class="snapimage">
+                <h1 class="tonic-text">Tonic</h1>
+                <ul class="exp-items">
+                    <li class="items canopy">CANOPY</li>
+                    <li class="items job-year"> Back End Dev</li>
+                    <li class="items job-year">2021</li>
+                </ul>
+                <p class="canopy-p">
+                    A daily section of privately personalized reads: no accounts or signups required.
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- Fourth Card -->
+
+        <div class="work-container">
+
+            <div class="project1">
+                <img src="images/multi-post-2.png" alt="Snapshoot Portfolio" class="snapimage">
+                <h1 class="tonic-text">Tonic</h1>
+                <ul class="exp-items">
+                    <li class="items canopy">CANOPY</li>
+                    <li class="items job-year"> Back End Dev</li>
+                    <li class="items job-year">2021</li>
+                </ul>
+                <p class="canopy-p">
+                    A daily section of privately personalized reads: no accounts or signups required.
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+
+        </div>
+
+        <!-- About Page -->
+
+        <section class="about-container">
```

Comment:
- [ ] To keep your webpage semantically correct, kindly take out the about-me section from the work section as they are two separated sections.
- [ ] Kindly align the elements of this section with flexbox.